### PR TITLE
LPS-176501 | Can load API Explorer from custom context

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Add a test case to load API Explorer from custom context

**Test Plan**
Given ROOT context changed to 'liferay'

When I navigate to localhost:8080/liferay/o/api

Then Liferay API Explorer loads correctly

**JIRA Link:**
https://issues.liferay.com/browse/LPS-176501

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Review by magjed4289 — APPROVED** [review]

File: build-test.xml [hunk 1]
```diff
@@ -14616,7 +14616,7 @@ mail.send.blacklist=</echo>
 						<replace
 							file="${app.server.dir}/conf/catalina.properties"
 							token="common.loader=&quot;${catalina.home}/webapps/ROOT/WEB-INF/lib/support-tomcat.jar&quot;"
-							value="common.loader=&quot;${catalina.home}/webapps/liferay/WEB-INF/lib/support-tomcat.jar&quot;"
+							value="common.loader=&quot;${catalina.home}/webapps/${portal.context}/WEB-INF/lib/support-tomcat.jar&quot;"
```
Comment (Hijky) [inline build-test.xml:14619]: This is the reason cause the word world doesn't work, I need to check the file carefully next time

Comment (Hijky) [reply build-test.xml:14619]: Thanks Meg !

**Review by yanan — APPROVED** [review]

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**